src/components/research-form-wizard/ResearchFormWizard.tsx

```diff
@@ -0,0 +1,161 @@
+import Image from 'next/image';
+import { useEffect, useState } from 'react';
+import Modal from 'react-bootstrap/Modal';
+import styled from 'styled-components';
+
+import { useCookiePreferences } from '@/hooks/useCookiePreferences';
+import { useResearchWizardEvents } from '@/hooks/useResearchWizardEvents';
+import { useResearchWizardStore } from '@/stores/researchWizard';
+import { recordResearchFromEvent } from '@/utils/analytics';
+
+import { StepLayout } from './StepLayout';
+import ThumbsUpIcon from './thumbs-up.svg';
+
+const MobileWrapper = styled.div`
+  width: 276px;
+  height: 72px;
+  position: fixed;
+  bottom: 20px;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: 10px 20px;
+  background-color: white;
+  color: #000000;
+  border: none;
+  border-radius: 100px;
+  cursor: pointer;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  display: flex;
+  align-items: center;
+  gap: 15px;
+`;
+
+const MobileFormWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const DesktopFormWrapper = styled.div`
+  position: fixed;
+  bottom: 1em;
+  right: 1em;
+  margin: 0 auto;
+`;
+
+const IconContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 10px;
+  border-radius: 50%;
+  width: 48px;
+  height: 48px;
+  color: white;
+`;
+
+export const ResearchFormWizard = () => {
+  const set = useResearchWizardStore((state) => state.set);
+  const [matches, setMatches] = useState(true);
+  const currentStepIndex = useResearchWizardStore((state) => state.currentStepIndex);
+  const formSteps = useResearchWizardStore((state) => state.formSteps);
+  const currentStepSubmission = useResearchWizardStore((state) => state.currentStepSubmission);
+  const cookieData = useCookiePreferences();
+  const { showMobileResearchForm, setShowMobileResearchForm, isResearchFormDismissed, setIsResearchFormDismissed } =
+    useResearchWizardEvents();
+  const [researchEventSent, setResearchEventSent] = useState(false);
+  const [currentStep, setCurrentStep] = useState<JSX.Element>(formSteps[currentStepIndex].component());
+
+  useEffect(() => {
+    setMatches(window.matchMedia('(min-width: 1120px)').matches);
+  }, []);
+
+  useEffect(() => {
+    window.matchMedia('(min-width: 1120px)').addEventListener('change', (e) => setMatches(e.matches));
+  }, []);
+
+  const dismissForm = () => {
+    recordResearchFromEvent('close-research-form', { questionNumber: currentStepIndex + 1 });
+    if (showMobileResearchForm) {
+      setShowMobileResearchForm(false);
+    }
+    setIsResearchFormDismissed(true);
+    localStorage.setItem('researchFormDismissed', 'true');
+  };
+
+  const handleFormButton = () => {
+    if (currentStepSubmission) {
+      recordResearchFromEvent('research-form-submit', {
```

Comment:
@jackson-harris-iii Let's use `q3_developer_survey` instead of the prefix `research-form`